cluster-scope/base/crds/clusterloggings.logging.openshift.io/crd.yaml

```diff
@@ -0,0 +1,36 @@
+---
+apiVersion: "logging.openshift.io/v1"
+kind: "ClusterLogging"
+metadata:
+  name: "instance"
+  namespace: "openshift-logging"
```

Comment:
@eranra this looks like a namespaced CR and not a cluster-scope CRD, I don't think this should be in base/crds/..

Reply:
@4n4nd yes, I think you're pointing to the same thing: https://github.com/operate-first/apps/pull/153#discussion_r564420099

Reply:
oh oops nvm